code/modules/clothing/shoes/magboots.dm

```diff
@@ -76,20 +76,33 @@
 
 /obj/item/clothing/shoes/magboots/advance
 	name = "advanced magboots"
-	desc = "Advanced magnetic boots that have a lighter magnetic pull, placing less burden on the wearer."
```

Comment:
This feels like a downgrade in terms of a description, from concise to vague and memey

Reply:
Modified.